_includes/manuals/2.4/1.2_release_notes.md

```diff
@@ -254,6 +254,20 @@ See the upgrade warnings for more information.
 
 Foreman 2.1 started to default to Puma while still allowing to fall back to Passenger. This support was kept in for users who ran into problems. Now that it has been the default for a few releases, this release deprecates Passenger support which [will be removed in the future](https://community.theforeman.org/t/drop-passenger-support/22787). Users are strongly encouraged to switch to Puma or reach out if there's something holding them back.
 
+#### Puppet ENC
+
+Puppet specific ENC functionality is deprecated and [will be moved to plugin in the future](https://community.theforeman.org/t/puppet-plugin-release-and-its-future/22335).
+
+Features to be removed include:
+* Environment
+* Puppetclass
+* ConfigGroup
+* SmartClassParameter
+
+Endpoint `/enc` will continue to provide information about the Host, but without the removed information.
+Users, who want to continue using these features, are advised to follow the puppet plugin stabilization and install it with the following version or report any issues holding them from doing so.
```

Comment:
```suggestion
Users who want to continue using these features, are advised to follow the Puppet plugin stabilization and install it with the following version or report any issues holding them from doing so.
```

I'm not sure what `following` means. Usually this refers to something later in the text. Do you mean `next Foreman version`?

Also, is there an issue tracking stablization?

Reply:
Nope, there is not an issue, there is not an RM project yet.

Reply:
If you create a project, I can add that there. I belive it is time for the plugin to move to Redmine anyway.

Reply:
Added the issue https://projects.theforeman.org/issues/32182 where we should fill all the necessary work to stabilize the plugin.